data/CollegeHero.sql

```diff
@@ -151,6 +151,42 @@ CREATE PROCEDURE createStaff
     INSERT INTO staff VALUES (NULL, name, password, department, staffType, phone);
   END//
 
+DROP PROCEDURE IF EXISTS getAllSectionInfoByClassID;
+CREATE PROCEDURE getAllSectionInfoByID(IN ID VARCHAR(45))
+    BEGIN
+        SELECT
+        class.cID,
```

Comment:
It looks like you're getting all attributes for class. You can instead just use `SELECT *` so you don't have to type out all of the attributes.